### PR TITLE
Launch up script update

### PR DESCRIPTION
**Pull request by BillyWooo**
Optimize the python launch script to wait for worker to start.

### CODE REVIEW

File: tee-worker/local-setup/launch.py [hunk 1]
```diff
@@ -58,19 +61,36 @@ def main(processes, config_path, parachain_type):
...
+                    break
+                if counter >= 60:
+                    print("Worker initialization error. Exit")
+                    return 0
+                counter = counter + 1
+
+        worker_i += 1
+
+    c.close()
+    print("Worker(s) started!")
```
Comment (Kailai-Wang) [inline tee-worker/local-setup/launch.py:105]: I think you need to print "workers started" until the last worker is started (currently it looks for `localhost:4545` which is the first worker)

So when I see this message, worker2 is not fully started, actually

File: tee-worker/local-setup/launch.py [hunk 2]
```diff
@@ -61,14 +61,15 @@ def main(processes, config_path, parachain_type):
...
+    UNTRUSTED_HTTP_PORT=4545
     c = pycurl.Curl()
-    c.setopt(pycurl.URL, 'http://localhost:4545/is_initialized')
-
     worker_i = 0
     worker_num = len(config["workers"])
     for w_conf in config["workers"]:
         processes.append(run_worker(w_conf, worker_i))
         print()
+        url = 'http://localhost:' + str(UNTRUSTED_HTTP_PORT+worker_i) + '/is_initialized'
```
Comment (Kailai-Wang) [inline tee-worker/local-setup/launch.py]: Actually the port should come from parsing the worker.config (I could use something other than 4545 due to e.g. port conflict), but let's leave it - it's for local use anyway

**Review by Kailai-Wang — APPROVED** [review]

File: tee-worker/local-setup/launch.py (hunk 2 above)
Comment (BillyWooo) [reply tee-worker/local-setup/launch.py]: Updated.

File: tee-worker/local-setup/launch.py (hunk 1 above)
Comment (BillyWooo) [reply tee-worker/local-setup/launch.py:105]: Fixed.

**Pull request merged**